### PR TITLE
feat(ButtonGroup): Add support for input+button combo

### LINKED ISSUE #322: Component: text field + combo button
We need Text Field + Combo box combined item for IFCExporter which will show up desired folder to user and allow him to navigate & select file with desired extension from user.

### PR DESCRIPTION
**Pull request by mayank99**
Added story for input+button combo.

image

Closes #322

## Checklist

- [ ] Add meaningful unit tests for your component (verify that all lines are covered)
- [ ] Verify that all existing tests pass
- [ ] Add component features demo in Storybook (different stories)
- [ ] Approve test images for new stories
- [ ] Add screenshots of the key elements of the component

### CODE REVIEW

File: stories/core/Buttons/ButtonGroup.stories.tsx [hunk 1]
```diff
@@ -119,3 +121,27 @@ Overflow.parameters = {
     ignoreElements: ['small'],
   } as CreeveyStoryParams,
 };
+
+export const InputButtonCombo: Story<ButtonGroupProps> = (args) => {
```
Comment (bentleyvk) [inline stories/core/Buttons/ButtonGroup.stories.tsx:125]: So you said you fixed it?
input-button-combo

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Did you run `yarn` and restart storybook?

Comment (bentleyvk) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Yup and ran on new folder freshly.

Comment (bentleyvk) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: image
It seems that there is your latest changes.

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: wow, now it's inconsistent. It worked for me, and then I refreshed and it broke, and then I refreshed again and it works.

This is a bug in storybook and I am not sure what to do.

Comment (bentleyvk) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Any idea which part of storybook is causing this?

Comment (saaaaaally) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Its working fine for me

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: > Its working fine for me

Does it consistently work fine? Try refreshing a few times and testing each time, and also try incognito mode

Comment (saaaaaally) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Oh yeah randomly stops working 😕

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Weirdly, it works fine in Firefox as well as Edge and Brave (both of which use Chromium). So maybe it's just a Chrome bug with positioned iframes.

Comment (FlyersPh9) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Was not able to reproduce this in the latest versions of Safari, Firefox, or Brave.  I am able to reproduce on the latest version of Chrome though (Version 96.0.4664.110 (Official Build) (x86_64)).

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: This PR should fix the issue: https://github.com/iTwin/iTwinUI/pull/422

Comment (mayank99) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Fixed, please check now. @bentleyvk @saaaaaally

Comment (saaaaaally) [reply stories/core/Buttons/ButtonGroup.stories.tsx:125]: Yes, its working now for me.

**Review by bentleyvk — APPROVED** [review]

**Pull request merged**